frontend/assets/icons/viewer/align.svg.tsx

```diff
@@ -0,0 +1,28 @@
+/**
```

Comment:
Clicking on a view with clipping plane should then allow me to click on the clip tool to edit the plane - at the moment the bottom tool bar doesn't seems to think there's a clip on the model
![24-01-04 02-28-33](https://github.com/3drepo/3drepo.io/assets/11945337/79a81c78-067f-4cfe-afe5-fe36c719cfcf)

Reply:
@The-Daniel  should not automatically enable the edit mode. It should just set the tool to be the state where there;s a clipping plane, but not in edit mode (i.e. same as you've set the clip, but clicked on the main clipping button to disable the editing)

Reply:
@The-Daniel  this was not addressed

> @The-Daniel should not automatically enable the edit mode. It should just set the tool to be the state where there;s a clipping plane, but not in edit mode (i.e. same as you've set the clip, but clicked on the main clipping button to disable the editing)

Reply:
Oops, I did fix that at some point but I think my other changes must have reverted it